### ISSUE TITLE
[Elastic Agent] elastic-agent script uses invalid command line args when installed with the DEB package

### ISSUE THREAD

**Issue by dolaru** [elastic/beats#20589]
## Environment
OS: **Ubuntu 18.04.5 LTS**
Agent version: **7.8.1**
Installation method: **DEB package**

## Issue
Calling `elastic-agent` fails with the following output:
```shell
$ elastic-agent --help
Error: unknown command "/usr/share/elastic-agent" for "elastic-agent"
Run 'elastic-agent --help' for usage.
unknown command "/usr/share/elastic-agent" for "elastic-agent"
```

Looks like the problem lies in`/usr/bin/elastic-agent`, where the flags are specified with a single `-` instead of `--`:
```sh
#!/usr/bin/env bash

# Script to run Elastic-Agent in foreground with the same path settings that
# the init script / systemd unit file would do.

exec /usr/share/elastic-agent/bin/elastic-agent \
  -path.home /usr/share/elastic-agent \
  -path.config /etc/elastic-agent \
  -path.data /var/lib/elastic-agent \
  -path.logs /var/log/elastic-agent \
  "$@"
```

After fixing the `-`, the script complains about the `--path.logs` flag: `unknown flag: --path.logs`

**Comment by ph** [comment]
@blakerouse I believe this is fixed for 7.9 can you verify?

**Comment by blakerouse** [comment]
Yes this is fixed in 7.9
